egs/fame/s5/run.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+stage=0
+feat_nj=60
+train_nj=60
+decode_nj=20
```

Comment:
I think feat_nj and train_nj is set up too high for 18 hours of speech data -- 12 or 16 jobs would be more appropriate.